user_guide/intro_company_structure_org_selector.rst

```diff
@@ -0,0 +1,198 @@
+- Eastern California Internet Freelance Employees
+
+
+The corporate structure is essential, as it defines the scope of the user’s tasks and and available resources. As a 
+business application, OroCRM provides tools to reflect the corporate structure normally.
+
+
+How Corporate Structure is Reflected in OroCRM
+----------------------------------------------
+
+Business units
+^^^^^^^^^^^^^^
+
+In OroCRM every business, division, or team of any kind may be represented by a business unit. The units can be 
+hierarchically organized in a tree-like structure that reflects the actual subordination of the organization.
+ 
+So, for example, to represent the hierarchy of the above-mentioned telecom in California, we need to create two business 
+units that correspond to the two offices *"Western California"* and *"Eastern California"*. Each of them will have child 
+business units that represent the directions: *"Stationary Phones"*, *"Mobiles"*, etc. Each of these child units, in its 
+turn, will be a parent unit to other business units that correspond to the teams.
+
+For example:
+
+* Eastern California business unit has a child business unit:
+
+  * Eastern California Stationary Phones that has child business units:
+
+    * Eastern California Stationary Phone Office Employees
+  
+    * Eastern California Stationary Phone Field Workers
+  
+    * Eastern California Stationary Phone Freelance Employees
+
+So, the system administrator usually creates such reflection of the organization’s hierarchy and defines what users 
+belong to them. *This affects the access and permissions granted to the users within the company*.
+ 
+For example, users assigned directly to the  Eastern California business unit may see all the orders, requests and 
+sales reported by any of the users of the unit and its child unit. Thus an office manager will see all the accounts 
+created in any direction and by any kind of employee.
+
+Users assigned to the child units may see only the accounts created within their unit - so, employees of the Stationary 
+Phone department will only see the items reported by members of their department, and won’t see items reported by 
+representatives of the Mobiles’ department. 
+
+In the same manner, the access may be limited for users assigned to the child-units of the next level, e.g. freelancer’s 
+may only see the information that has been reported or assigned to them directly, and office employees will see 
+information reported by any of their co-workers.
+
+This will protect information on one hand, and help to protect users from being overwhelmed with non-relevant 
+information. 
+
+Please refer to the :ref:`Access and permissions guide <user-guide-user-management-permissions-basic>` for more details.
+
+.. hint::
+
+    By the user name, you can see the name of the business unit, that *"owns"* the user. This is also defined by the 
+    system administrator and serves as additional information for other users.
```

Comment:
Edit: Next to the user name, you can see the name of the business unit that *"owns"* the user. This is also defined by the system administrator and serves as additional information for other users.